wagtail/test/testapp/migrations/0015_gallerypage_gallerypageimage.py

```diff
@@ -66,6 +66,15 @@ class Migration(migrations.Migration):
                         to="tests.gallerypage",
                     ),
                 ),
+                (
+                    "exclude_field",
+                    modelcluster.fields.ParentalKey(
```

Comment:
Two ParentalKeys on the same model isn't really valid, because ParentalKey means "this model will only ever be edited as a child of the linked model". The test here isn't creating any GalleryPageImage instances (which it really should be doing if it's going to test that `exclude_fields_in_copy` is doing the right thing, rather than just confirming that it doesn't crash), but if it did, we'd have problems with the `page` key being non-null. It would be better to set up a dedicated child model for `PageWithExcludedCopyField` here.